### DISCUSSION TITLE
Add docs about how to use unix socket

### DISCUSSION THREAD

**Discussion by frederikhors** [ory/kratos#1874] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

I saw this PR https://github.com/ory/kratos/pull/1255 for unix sockets support but there is no docs on how to use that.

**Describe the solution you'd like**

I would like to read how to configure my client (Golang) to use this socket from my Golang application.

Right now I'm using:

```go
kratosConfiguration := kratos.NewConfiguration()

cookie := getCookie()

session, _, err := kratosPublic.V0alpha1Api.ToSession(context.Background()).Cookie(cookie).Execute()
if err != nil {
  http.Error(w, "", http.StatusUnauthorized)

return fmt.Errorf("unauthorized user or malformed session cookie: %w", err)
}
```

How to use unix socket here?

**Comment by aeneasr** [comment, marked as answer]
Oh, I don't think the SDKs support socket connectivity :/
